x/gov/migrations/v3/store_test.go

```diff
@@ -1,4 +1,4 @@
-package v046_test
+package v3_test
```

Comment:
So v046 == v3, and this PR should create v4, and a v3->v4 migration. Is that correct?